Src/PerformanceCollector/Perf.Shared/Implementation/PerformanceCounterUtility.cs

```diff
@@ -58,6 +68,100 @@ public static string FormatPerformanceCounter(PerformanceCounter pc)
         }
 #endif
 
+        public static bool IsPerfCounterSupported()
+        {
+            if (IsWebAppRunningInAzure())
+            {
+                return true;
+            }
+            else
+            {
+#if NET45
+                return true;
+#elif NETSTANDARD1_6
+                return false;
+#elif NETSTANDARD2_0
+                if (IsWindows)
+                {
+                    return true;
+                }
+                else
+                {
+                    // return false;
+                    return true;
+                }
+#endif
+            }
+        }
+
+#if NETSTANDARD1_6
```

Comment:
nit: I see that for the method `GetPerformanceCollector`, NETSTANDARD1.6 has it's own method, but NET45 and NETSTANDARD2.0 are sharing a method.

I would expect that either all three have separate methods or all three share the same method.

Reply:
right. moving all to separate methods. its much easier to read now.